docs/docs/configuration/index.md

```diff
@@ -3,9 +3,11 @@ id: index
 title: Frigate Configuration
 ---
 
-For Home Assistant Addon installations, the config file needs to be in the root of your Home Assistant config directory (same location as `configuration.yaml`). It can be named `frigate.yaml` or `frigate.yml`, but if both files exist `frigate.yaml` will be preferred and `frigate.yml` will be ignored.
+For Home Assistant Add-on installations, the config file should be at `/addon_configs/ccab4aaf_frigate/config.yaml` ([click here to learn more on how to access this directory](#accessing-add-on-config)).
```

Comment:
Might want to change this or clarify the directory name for the add-on variants (plus anywhere else you use this in your changes).

Reply:
This is clarified in the "click here" link. You think it's not enough?

Reply:
Maybe I'd change the directory name in the example to `/addon_configs/<addon_directory>/config.yaml` and then say something like "where `<addon_directory>` is specific to the version of the Add-on you are running. See the list of directories [here](#accessing-add-on-config)" or something similar.

Reply:
Sounds good. Thank you.

Reply:
@hawkeye217, made the changes. Please let me know if you liked it. :)